doc/developer/platform/ux.md

```diff
@@ -299,17 +299,63 @@ The HTTPS server exposes two APIs:
     additional control over query execution and result fetching, a la the
     [Snowflake SQL REST API].
 
-    TODO: fully describe the output format.
+    ### Output format
 
-    TODO: design support for binding parameters.
+    The output format is a JSON object with one key, `results`, whose values is
+    an array of serialized `adapter::command::SimpleResult`, representing all of
+    the statements' executions.
+
+    The serialized `adapter::command::SimpleResult` values are representing in JSON as:
+
+    `SimpleResult` value | JSON value
+    ---------------------|------------
+    `Rows` | `{"rows": <2D array of JSON-ified results>, "col_names": <array of text>}`
+    `Error` | `{"error": <Error string from execution>}`
+    `Ok` | `{ "ok": <tag>}`
+
+    ### Binding parameters
+
+    You can include parameters alongside the query by including on the query a
```

Comment:
This contradicts how queries are sent above:

> The request body specifies the SQL string to execute.

Do we want both? In either case, we should specify the json format fully and say that it should be the request body.

Reply:
Attempted to clarify, but I think the majority of the issue here was imprecision in my language. lmk if this is still unclear and will gladly accept suggestions/edification.